proposals/0011-inline-spirv.md

```diff
@@ -0,0 +1,347 @@
+<!-- {% raw %} -->
+
+# Inline SPIR-V
+
+## Instructions
+
+> This template wraps at 80-columns. You don't need to match that wrapping, but
+> having some consistent column wrapping makes it easier to view diffs on
+> GitHub's review UI. Please wrap your lines to make it easier to review.
+
+> When filling out the template below for a new feature proposal, please do the
+> following first:
+
+> 1.  exclude the "Planned Version", "PRs" and "Issues" from the header.
+> 2.  Do not spend time writing the "Detailed design" until the feature has been
+>     merged in the "Under Consideration" phase.
+> 3.  Delete this Instructions section including the line below.
+
+--------------------------------------------------------------------------------
+
+*   Proposal: [0011](0011-inline-spirv.md)
+*   Author(s): [Steven Perron](https://github.com/s-perron)
+*   Sponsor: TBD
+*   Status: **Under Consideration**
+
+*During the review process, add the following fields as needed:*
+
+*   Planned Version: 20YY
+*   PRs: [#NNNN](https://github.com/microsoft/DirectXShaderCompiler/pull/NNNN)
+*   Issues:
+    [#NNNN](https://github.com/microsoft/DirectXShaderCompiler/issues/NNNN)
+
+## Introduction
+
+Define a set of Vulkan specific builtin functions, types, and attributes that
+will allow vendors to implement their extensions. The vendors should be able to
+write a self-contained header file that users can use without knowing about
+inline SPIR-V.
+
+## Motivation
+
+In Vulkan, there is a culture of allowing a vendor to define extensions that are
+only interesting to them. From the perspective of a driver this is acceptable
+because company B does not have to support the extension put out by vendor B, if
```

Comment:
"company A does not have to support the extension put out by vendor B"?

Reply:
```suggestion
because company A does not have to support the extension put out by vendor B, if
```